### PR TITLE
fix(tuner): logging

### PR DESCRIPTION
**Pull request by tadejsv**
This makes logging play nicer with the progress bar - it is not interrupted, logging instead is shown above it.

I also remove any dependency on jina logger.

The new logging setup is rich-compatible, and tries to be non-intrusive - only the settings for `finetuner` logger are changed, root and other loggers are not affected.

Here's how the new logging looks like
logg_new

### CODE REVIEW

File: finetuner/toydata.py [hunk 1]
```diff
@@ -86,24 +86,21 @@ def _download_qa_data(
...
-            if not os.path.exists(v['filename']):
-                urllib.request.urlretrieve(
-                    v['url'], v['filename'], reporthook=lambda *x: t.update(0.01)
-                )
-
-        with open(targets['covid-csv']['filename']) as fp:
-            lines = csv.DictReader(fp)
-            for idx, value in enumerate(lines):
-                if is_testset is None:
+    for k, v in track(targets.items(), 'Downloading chatbot data'):
```
Comment (JoanFM) [inline finetuner/toydata.py:89]: ignore k maybe?

Comment (tadejsv) [reply finetuner/toydata.py:89]: sure

**Comment by bwanglzu** [comment]
does it work on notebook?

File: finetuner/tuner/callback/early_stopping.py [hunk 2]
```diff
@@ -23,6 +23,7 @@ def __init__(
         patience: int = 2,
         min_delta: int = 0,
         baseline: Optional[float] = None,
+        verbose: bool = False,
```
Comment (bwanglzu) [inline finetuner/tuner/callback/early_stopping.py:26]: docstring is not updated here

**Review by bwanglzu — CHANGES_REQUESTED** [review]
minor comments

**Comment by tadejsv** [comment]
@bwanglzu Works similar as before, here's what I see at the end of training. The only thing is that there is still the annoying "blinking" warning message, but that's the same as before.

image

**Review by bwanglzu — APPROVED** [review]
LGTM!

**Pull request merged**